spec/features/create_peep_spec.rb

```diff
@@ -0,0 +1,19 @@
+feature 'creating a peep' do
+  scenario 'requires sign in' do
+    visit '/peeps/new'
+    expect(page).to have_content('Sign in or Sign up')
+  end
+
+  let (:user) do
+    User.create(name: 'Ivan Sathianathan', username: 'ivan', email: 'ivan@test.com', password: '12345', password_confirmation: '12345')
```

Comment:
Metrics/LineLength: Line is too long. [135/90] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)